### DISCUSSION TITLE
Welcome to IDBacApp Discussions!

### DISCUSSION THREAD

**Discussion by chasemc** [chasemc/IDBacApp#145] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ERBringHorvath** [comment]
Hi Chase--I just had a quick question about the MALDI-MS protein fingerprint IDBac uses to construct the dendrogram in-app. Is there a method to assign organism identification using the mzXML data, or would we need to separately sequence 16S ribosomal DNA to attach any taxonomic identifiers? 

Thanks!

**Reply by chasemc** [reply]
You can add whatever ids you like within the app, (no good way to do that with mzxml)

See "Figure 6: Input sample information." and related text in https://www.jove.com/t/59219/using-open-source-maldi-tof-ms-idbac-pipeline-for-analysis-microbial

**Reply by chasemc** [reply]
I may have misinterpreted your question.  Are you asking if there is a database to search against to ID your strains?

**Comment by ERBringHorvath** [comment]
Yes. In the IDBac pub, you compare the MALDI-MS protein fingerprint
dendrogram to one made using 16S sequences and determine that the protein
fingerprint is nearly as accurate in grouping organisms by clade as the 16S
sequence. So, I was wondering if there was a way to use the mzML/mzXML data
to obtain a degree of taxonomic classification, similar to how I might do
it with 16S sequence data. I'm working with ~400 samples, so I can go back
to the source plates and isolate select strains for 16S sequencing if I
have to, but I was just curious if there was a method available using the
data I already have.

I appreciate your help. I'm sure you don't remember, but I attended the
IDBac workshop at ASP in South Carolina this past summer. I'm a member of
Jackie Winter's lab at the University of Utah.

Best,
Eli

*Eli Bring Horvath*
University of Utah
PhD Candidate, Department of Medicinal Chemistry

**Reply by chasemc** [reply]
>I'm working with ~400 samples, so I can go back
to the source plates and isolate select strains for 16S sequencing if I
have to

That is how we have been doing it while building up enough knowns. The online spectrum database of known strains is actively being developed and should cover a decent taxonomic spread, but isn't quite public yet; your use case is exactly what it is being developed for.

I'll pass your message along to see if there's enough setup right now or some cleaned data we can send over to help.
